src/compile/mark/encode/position-rect.ts

```diff
@@ -215,15 +224,7 @@ function positionAndSize(
     stack,
     offset,
     defaultRef: pointPositionDefaultRef({model, defaultPos: 'mid', channel, scaleName, scale}),
-    bandPosition: center
-      ? offsetType === 'encoding'
-        ? 0
-        : 0.5
-      : isSignalRef(bandSize)
-      ? {signal: `(1-${bandSize})/2`}
-      : isRelativeBandSize(bandSize)
-      ? (1 - bandSize.band) / 2
-      : 0
+    bandPosition: offsetType === 'encoding' ? 0 : bandPosition
```

Comment:
This is not consistent with the old version. 
The `bandPosition` used by `defaultRef` is `signal-relative` value in the old implementation, but `0` in the new implementation, when center is `false` and `offsetType` check is `true`.